src/lib/components/Header/_styles.scss

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/global';
+@import '../../../styles/tools/breakpoints';
+@import '../../../styles/tools/mixins';
+
+web-header:not(:defined) {
```

Comment:
does this work in a polyfilled universe?

Reply:
probably not. I also think that any FOUC with this header is bad so I'm going to move all of its guts to the light dom